components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -97,6 +119,22 @@ An [environment]({{< relref "environments" >}}) can be used to map an organizati
 
 Chef Infra Clients provide secure API access to the Chef Infra Server.
 
+The Chef Infra Server UI let's you access the _Clients_ object in the following ways:
+
+- All the _Clients_ of an organization can be listed at one place.
+
+- You can create a client using the button as shown below:
+
+{{< figure src="/images/automate/create-client-button.png" alt="Create Client Button">}}
+
+- In the above image, the search bar stating __Search Clients__ can be used to search a specific client from the list.
```

Comment:
I understand what you're trying to do with this list, but you're mixing conceptual and procedural content.

Conceptual:
The _Chef Infra Server_ tab lets you:
- Create a client
- Search for one client from a list of clients
- Delete a client
- View the details of a client
- Reset a client public key.

Then give the procedural steps for these actions. The sequence is "tell them what you are going to tell them how to do and then tell them how to do it."